.github/workflows/ci.yml

```diff
@@ -45,7 +45,7 @@ jobs:
           python-version: ${{ matrix.python-version }}
 
       - name: Install/Upgrade Python dependencies
-        run: python -m pip install --upgrade pip wheel
```

Comment:
Do we need this now that we've specified the setuptools requirements in our setup.py?

Reply:
yes. I'm testing with py3.8 on my machine, and if I keep the pip which is shipped by default it fails with an internal `AssertionError` :facepalm: